Day 1 - Any two numbers from the list add up to K

```diff
@@ -0,0 +1,39 @@
+Question :
+
+Given a list of numbers and a number k, return whether any two numbers from the list add up to k.
+For example, given [10, 15, 3, 7] and k of 17, return true since 10 + 7 is 17.
+
+public class addition_two_number {
+
+    static boolean Logic(String[] split, Long value) {
+        String s = Arrays.toString(split);
+        String temp1 = "", temp2 = "",temp3="";
+        for (int i = 0; i < split.length; i++) { 
+            value = value - Long.parseLong(split[i]);  // Subtract each element with k and check the result element present in the array
+            temp1 = ", " + value + "]";
+            temp2 = ", " + value + ",";
+            temp3= "["+value+"]";
+            if (s.contains(temp1) || s.contains(temp2)) { //checking the result element with comma as it was strings
```

Comment:
What is the complexity of contains function here, Is this calculated for time complexity.